### PR TITLE
New audio annotator component

### PR DESCRIPTION
**Pull request by rvovard**
First version with following features:

- Load a dummy sound file
- Play / Pause
- Empty canvas with progress bar
- Timestamps

### CODE REVIEW

File: src/AudioPlayer.js [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+      this.props.onEnded(e);
+    });
+
+    // When the user pauses playback
+    audio.addEventListener('pause', (e) => {
+      this.clearListenTrack();
+      this.props.onPause(e);
+    });
+
+    // When the user drahs the time indicator to a new time
```
Comment (ixio) [inline src/AudioPlayer.js]: typo "drags"

File: src/App.js [hunk 2]
```diff
@@ -92,6 +93,7 @@ class App extends Component<void, AppState> {
         <Router>
           <Switch>
             <Route path='/audio-annotator/:annotation_task_id' render={route_props => <LegacyAudioAnnotator app_token={this.state.app_token} {...route_props} />} />
```
Comment (ixio) [inline src/App.js]: Old path should become legacy-audio-annotator and new one audio-annotator, like in FeatureService

File: src/AnnotationTaskList.js [hunk 3]
```diff
@@ -74,7 +74,7 @@ class AnnotationTaskList extends Component<AnnotationTaskListProps, AnnotationTa
           <td>{start_date.toLocaleDateString()}</td>
           <td>{diff_time.toUTCString().split(' ')[4]}</td>
           <td>{status_names[annotation_task.status]}</td>
-          <td><Link to={'/audio-annotator/' + annotation_task.id}>Task link</Link></td>
+          <td><Link to={'/annotator/' + annotation_task.id}>Task link</Link></td>
```
Comment (ixio) [inline src/AnnotationTaskList.js]: This should stay audio-annotator (cf App.js)

File: src/AudioPlayer.js (hunk 1 above)
Comment (rvovard) [reply src/AudioPlayer.js]: Corrected in commit 52fe109

File: src/App.js (hunk 2 above)
Comment (rvovard) [reply src/App.js]: Corrected as `/audio-annotator` and `/audio-annotator/legacy` in commit 52fe109

File: src/AnnotationTaskList.js (hunk 3 above)
Comment (rvovard) [reply src/AnnotationTaskList.js]: Corrected in commit 52fe109

**Pull request merged**